shell/app/modules/runtime/pages/overview/components/service-dropdown.tsx

```diff
@@ -81,11 +81,12 @@ const ServiceDropdown = (props: IProps) => {
     const warningMsg = i18n.t('runtime:deploying, please operate later');
     const envKey = (env || '').toLowerCase();
 
-    const vipContent = (
+    const vipContent = !isEmpty(addrs)
+      ? (
       <List
```

Comment:
should the `List` component has more indents according to the brackets around it?